script/airdrop-monitor.sh

```diff
@@ -0,0 +1,63 @@
+# This script monitors the Airdrop canister for new parcipants list,
+# and sends ethereum transaction to distribute ICP ERC20 tokens.
+#
+# Environment variables must be set:
+#
+# ETH_RPC_URL   RPC provider's URL
+#
+# ETH_FROM      The address to sign & send airdrop transactions.
+#               It is also the owner of Airdrop contract.
+#               Lastly seth must import the private key first.
+#
+# AIRDROP_FROM  The address of the Airdrop contract
+
+AIRDROP_CLAIMED_JSON="airdrop_claimed.json"
+AIRDROP_CLAIMED_HASH="airdrop_claimed.hash"
+INTERVAL_SECS=60
+
+test -z "$ETH_RPC_URL" && echo "Must set ETH_RPC_URL" && exit 1
+test -z "$ETH_FROM" && echo "Must set ETH_FROM" && exit 1
+test -z "$AIRDROP_FROM" && echo "Must set AIRDROP_FROM" && exit 1
+
+function sync_participant_list() {
```

Comment:
So this function will fetch the append-only list of addresses from the canister and then filter out those from airdrop_claimed.json right?

Reply:
Yes, that is what is planned.